.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+      ecmaVersion: 2020,
+      sourceType: "module"
+    },
+    extends: [
+      "plugin:@typescript-eslint/recommended",
+      "prettier/@typescript-eslint",
+      "plugin:prettier/recommended"
+    ],
+    rules: {
+      // Specify rules here
+    }
+  };
```

Comment:
all new line at the end and add it to the rule
not sure if it goes in eslintrc or prettierrc : P